From 1c21dada05770fafac2d8ca53b90db83327af1d5 Mon Sep 17 00:00:00 2001
From: Tom Rondeau <tom@trondeau.com>
Date: Mon, 21 Apr 2014 18:34:24 -0400
Subject: [PATCH 21/33] docs: doxygen mangles sections named with the same
 reference name. ControlPort and Vocoder both used
 'using' - renamed for each.

---
 docs/doxygen/other/ctrlport.dox |   12 ++++++------
 gr-vocoder/doc/vocoder.dox      |    2 +-
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/docs/doxygen/other/ctrlport.dox b/docs/doxygen/other/ctrlport.dox
index 9679036..a1ffcad 100644
--- a/docs/doxygen/other/ctrlport.dox
+++ b/docs/doxygen/other/ctrlport.dox
@@ -69,7 +69,7 @@ configure ICE can be found here:
 http://doc.zeroc.com/display/Ice/Properties+and+Configuration
 
 
-\section using Using ControlPort to Export Variables
+\section using_ctrlport Using ControlPort to Export Variables
 
 The ability to export variables from a block is inherited from
 gr::block. Then, when the flowgraph is started, the function
@@ -122,7 +122,7 @@ namespace gr {
       sync_bloc(....),
       d_a(a), d_b(b)
     { }
-    
+
     foo_impl::~foo_impl()
     { }
 
@@ -215,9 +215,9 @@ gets. These take the form:
   rpcbasic_register_get(const std::string& name,
 			const char* functionbase,
                         T* obj,
-			Tfrom (T::*function)(), 
+			Tfrom (T::*function)(),
 			const pmt::pmt_t &min, const pmt::pmt_t &max, const pmt::pmt_t &def,
-			const char* units_ = "", 
+			const char* units_ = "",
 			const char* desc_ = "",
 			priv_lvl_t minpriv_ = RPC_PRIVLVL_MIN,
 			DisplayType display_ = DISPNULL)
@@ -225,9 +225,9 @@ gets. These take the form:
   rpcbasic_register_set(const std::string& name,
 			const char* functionbase,
                         T* obj,
-			void (T::*function)(Tto), 
+			void (T::*function)(Tto),
 			const pmt::pmt_t &min, const pmt::pmt_t &max, const pmt::pmt_t &def,
-			const char* units_ = "", 
+			const char* units_ = "",
 			const char* desc_ = "",
 			priv_lvl_t minpriv_ = RPC_PRIVLVL_MIN,
 			DisplayType display_ = DISPNULL)
diff --git a/gr-vocoder/doc/vocoder.dox b/gr-vocoder/doc/vocoder.dox
index 0e12566..e3be774 100644
--- a/gr-vocoder/doc/vocoder.dox
+++ b/gr-vocoder/doc/vocoder.dox
@@ -20,7 +20,7 @@ by using:
     help(vocoder)
 \endcode
 
-\section using Using the vocoders
+\section using_vocoders Using the vocoders
 
 Note that most vocoders use short inputs instead of floats. This means you
 must convert audio signals from float to short by using a type conversion
-- 
1.7.10.4

